lib/ProgressPlugin.js

```diff
@@ -19,6 +19,7 @@ const createDefaultHandler = profile => {
 	const defaultHandler = (percentage, msg, ...args) => {
 		let state = msg;
 		const details = args;
+		const maxLength = Math.min(process.stderr.columns, 40);
```

Comment:
I think the 40 char restriction can be increased. It was a kind of workaround for this problem.

```suggestion
		const maxLength = Math.min(process.stderr.columns || Infinity, 40);
```

`Math.min(undefined, 40) => NaN`